### PR TITLE
CR-1050747 health thread for xilinx_u50_xdma_201920_2 and xilinx_u50_…

### PR DESCRIPTION
**Pull request by xdavidz**
this is for CR CR-1050747

also address the name change for vbnv as xilinx_u50_xdma_gen3x4_201920_3

### CODE REVIEW

File: src/runtime_src/core/pcie/driver/linux/xocl/subdev/feature_rom.c [hunk 1]
```diff
@@ -425,10 +425,10 @@ static void platform_type_append(char *prefix, u32 platform_type)
 		type = "_Recovery BLP";
```
Comment (houlz0507) [inline src/runtime_src/core/pcie/driver/linux/xocl/subdev/feature_rom.c]: maybe remove " BLP" which is confusing.

Comment (xdavidz) [reply src/runtime_src/core/pcie/driver/linux/xocl/subdev/feature_rom.c]: done

**Comment by xdavidz** [comment]
Neil has additional comments in CR: http://jira.xilinx.com/browse/CR-1050747
For maximum value 6400, we can validate and print different dmesg.

File: src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c [hunk 2]
```diff
@@ -59,9 +59,12 @@ MODULE_PARM_DESC(minimum_initialization,
 
 #define	MAX_DYN_SUBDEV		1024
 
+#define	CLK_MAX_VALUE		6400
 #define	CLK_SHUTDOWN_BIT	0x1
 #define	DEBUG_CLK_SHUTDOWN_BIT	0x2
 #define	VALID_CLKSHUTDOWN_BITS	(CLK_SHUTDOWN_BIT|DEBUG_CLK_SHUTDOWN_BIT)
+#define CLK_THROTTLED(clock, highoff, lowoff)		\
```
Comment (chienwei-lan) [inline src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c]: indent: tab

Comment (xdavidz) [reply src/runtime_src/core/pcie/driver/linux/xocl/mgmtpf/mgmt-core.c]: adjust to bit-field structure, the ioread32 should have done the endian convert.

**Pull request merged**